src/main/java/nextstep/subway/favorite/ui/FavoriteController.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.subway.favorite.ui;
+
+import nextstep.subway.auth.domain.AuthenticationPrincipal;
+import nextstep.subway.auth.domain.LoginMember;
+import nextstep.subway.favorite.application.FavoriteService;
+import nextstep.subway.favorite.domain.Favorite;
+import nextstep.subway.favorite.dto.FavoriteRequest;
+import nextstep.subway.favorite.dto.FavoriteResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequestMapping("/favorites")
+public class FavoriteController {
+  private FavoriteService favoriteService;
+
+  public FavoriteController(FavoriteService favoriteService) {
+    this.favoriteService = favoriteService;
+  }
+
+  @PostMapping
+  public ResponseEntity createFavorite(@AuthenticationPrincipal LoginMember loginMember, @RequestBody FavoriteRequest request) {
```

Comment:
```suggestion
  public ResponseEntity<Void> createFavorite(@AuthenticationPrincipal LoginMember loginMember, @RequestBody FavoriteRequest request) {
```

이런식으로 return body 가 없더라도 명시해주는게 좋을 것 같습니다.

Reply:
아하 오... 알겠습니다! 감사합니다!